frontend/src/features/org/org-status-banner.ts

```diff
@@ -0,0 +1,238 @@
+          );
+          return {
+            title:
+              daysDiff > 1
+                ? msg(
+                    str`Your org will be deleted in
+              ${daysDiff} days`,
+                  )
+                : `Your org will be deleted within one day`,
+            detail: html`
+              <p>
+                ${msg(
+                  str`Your subscription ends on ${formatISODateString(
+                    subscription!.futureCancelDate!,
+                    {
+                      month: "long",
+                      day: "numeric",
+                      year: "numeric",
+                      hour: "numeric",
+                    },
+                  )}. Your user account, org, and all associated data will be deleted.`,
+                )}
+              </p>
+              <p>
+                ${msg(
+                  html`We suggest downloading your archived items before they
+                  are deleted. To keep your plan and data, see
+                  ${billingTabLink}.`,
+                )}
+              </p>
+            `,
+          };
+        },
+      },
+      {
+        test: () =>
+          !readOnly && readOnlyOnCancel && !!subscription?.futureCancelDate,
+        persist: true,
+        content: () => {
+          const daysDiff = differenceInDays(
+            new Date(),
+            new Date(subscription!.futureCancelDate!),
+          );
+          return {
+            title:
+              daysDiff > 1
+                ? msg(str`Your org will be deactivated in ${daysDiff} days`)
+                : msg("Your org will be deactivated within one day"),
+            detail: html`
+              <p>
+                ${msg(
+                  str`Your subscription ends on ${formatISODateString(
+                    subscription!.futureCancelDate!,
+                    {
+                      month: "long",
+                      day: "numeric",
+                      year: "numeric",
+                      hour: "numeric",
+                    },
+                  )}. You will no longer be able to run crawls, upload files, create browser profiles, or create collections.`,
+                )}
+              </p>
+              <p>
+                ${msg(
+                  html`To keep your plan and continue crawling, see
+                  ${billingTabLink}.`,
+                )}
+              </p>
+            `,
+          };
+        },
+      },
+      {
+        test: () =>
+          !!readOnly && readOnlyReason === OrgReadOnlyReason.SubscriptionPaused,
+        persist: true,
+        content: () => ({
+          title: msg(str`Your org has been deactivated`),
+          detail: msg(
+            html`Your subscription has been paused due to payment failure.
+            Please go to ${billingTabLink} to update your payment method.`,
+          ),
+        }),
+      },
+      {
+        test: () =>
+          !!readOnly && readOnlyReason === OrgReadOnlyReason.SubscriptionPaused,
```

Comment:
```suggestion
          !!readOnly && readOnlyReason === OrgReadOnlyReason.SubscriptionCancelled,
```
Maybe just a copy-paste error? :)